OpenRA.Mods.Common/Widgets/Logic/Ingame/CommandBarLogic.cs

```diff
@@ -201,6 +201,23 @@ public CommandBarLogic(Widget widget, World world, Dictionary<string, MiniYaml>
 						world.OrderGenerator = new ForceModifiersOrderGenerator(Modifiers.Shift, false);
 				};
 			}
+
+			var keyOverrides = widget.GetOrNull<LogicKeyListenerWidget>("MODIFIER_OVERRIDES");
+			if (keyOverrides != null)
+			{
+				keyOverrides.OnKeyPress = ki =>
+				{
+					// HACK: enable attack move to be triggered if the ctrl key is pressed
```

Comment:
Note: this will be fixed by the next phase of my hotkey rework, but that is currently blocked on #13781, #13966, #13974 and in any case won't help on the prep branch.